### PR TITLE
New package: lhasa-0.3.1

### PR DESCRIPTION
**Pull request by mahiuchun**
#### General
- [X] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [X] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/lhasa/template [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Template file for 'lhasa'
+pkgname=lhasa
+version=0.3.1
+revision=1
+build_style=gnu-configure
+hostmakedepends="automake libtool"
+depends="liblhasa>=${version}_${revision}"
+checkdepends="tzdata"
```
Comment (ericonr) [inline srcpkgs/lhasa/template:8]: We added `tzdata` to the base system, this just needs to be propagated to CI. Even removing it from here, you should be able to run `./xbps-src -Q pkg lhasa` locally with success.

Comment (mahiuchun) [reply srcpkgs/lhasa/template:8]: I tried that after git pull and rebasing to master but I got same error as the original commit.

File: srcpkgs/lhasa/template [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+short_desc="Free Software LHA implementation"
+maintainer="Hill Ma <maahiuzeon@gmail.com>"
+license="ISC"
+homepage="https://github.com/fragglet/lhasa"
+distfiles="https://github.com/fragglet/lhasa/releases/download/v${version}/lhasa-${version}.tar.gz"
+checksum=3fb08e5f85a9b9dd023922896be9157d5fb5c0448424681810aaa2b0558a5f24
+
+if [ "$CROSS_BUILD" ]; then
+	configure_args+=" --host=${XBPS_CROSS_TRIPLET}"
+fi
```
Comment (ericonr) [inline srcpkgs/lhasa/template:18]: Are you sure this is necessary? We do it in the build style already...

**Review by ericonr — COMMENTED** [review]
Upstream seems dead, which is unfortunate. Do you have examples where lha archives are used?

**Comment by mahiuchun** [comment]
It is an optional runtime dependency for `binwalk`. For example, Award BIOS files use LHA compression and can be decompressed by `binwalk` (or manually) with this package installed.

File: srcpkgs/lhasa/template (hunk 1 above)
Comment (Chocimier) [reply srcpkgs/lhasa/template:8]: To be fixed by #31641

Comment (sgn) [reply srcpkgs/lhasa/template:8]: You need a `./xbps-src bootstrap-update`

File: srcpkgs/lhasa/template (hunk 2 above)
Comment (mahiuchun) [reply srcpkgs/lhasa/template]: Removed in 3cfe609. Not sure why all the native builds fail now :(

**Comment by ericonr** [comment]
Please rebase to fix conflict and consider not splitting `liblhasa` into its own package, just leaving the CLI utility together with the library and a separate `-devel` one :)

**Comment by mahiuchun** [comment]
Done

**Pull request merged**